frontend/lib/pages/food_item_page.dart

```diff
@@ -47,8 +47,9 @@ class _FoodItemPageState extends State<FoodItemPage> {
     );
   }
 
+ //TODO FIX THAT.
   Future<void> createFood() async {
-    int code = await server.createFood(this.food);
+    int code = await server.addFood(this.food, -1);
```

Comment:
looking great so far :) awesome job! Just a quick question, what is the TODO fix that? And also in the addFood method, you have a -1 for the foodCollectId? Is there a reason for this?

Reply:
yeah so addFood takes two parameters the food object and the id of the food collect. we haven't implemented the foodCollect page yet so we hardcoded the id of the foodcollect to -1 for now. once we implement the food collect page we'll switch the id to be dynamic

Reply:
Okay I see, if you can create a task in the trello board as a reminder to fix this, that would be great :)